superset/connectors/druid/models.py

```diff
@@ -988,11 +988,9 @@ def values_for_column(self, column_name: str, limit: int = 10000) -> List:
     def get_query_str(self, query_obj, phase=1, client=None):
         return self.run_query(client=client, phase=phase, **query_obj)
 
-    def _add_filter_from_pre_query_data(
-        self, df: Optional[pd.DataFrame], dimensions, dim_filter
-    ):
+    def _add_filter_from_pre_query_data(self, df: pd.DataFrame, dimensions, dim_filter):
```

Comment:
The return type from `client.export_pandas()` is never `None` per [here](https://github.com/druid-io/pydruid/blob/80cf6fe767817c8c30e908ea2e145bee09aa6d9b/pydruid/query.py#L202).